culture_day_lesson_d.md

```diff
@@ -1,35 +1,42 @@
-# Culture Day Lesson D: Interview with People in Technology
+# Culture day: Interview people in technology fields
 
 ## Learning Objectives
 
 Students will be able to...
 
-* Identify the people in their family, or community, who have technology-related jobs
-* Create interview questions to find out about their jobs
-* Conduct an interview
-* Identify the different types of roles and skills needed in the technology industry
-* Identify technology skills needed in non-technology organizations
-* Reflect on how technology could be part of their future careers
+* Identify the people in their family, or community, who have technology-related jobs.
+* Create interview questions to find out about their jobs.
+* Conduct an interview.
+* Identify the different types of roles and skills needed in the technology industry.
+* Identify technology skills needed in non-technology organizations.
+* Reflect on how technology could be part of their future careers.
 
-### Emphasize with students
+### Explain to students
 
-#### Curriculum Competencies - Applied Skills and Technology
+You may come with stereotypes and myths about technology jobs. There are many different types of roles and skills needed in the technology industry. There are many roles that don't require you to sit at your computer all day long and write code.  
 
-Students may come with stereotypes and myths about technology jobs.  
+Here are a few examples:
 
-There are many different types of roles and skills needed in the technology industry.  Not everyone is a coder; and not everyone is coding all day long.  There are artistic designers, music creators, script writers, project managers, marketing specialists, quality assurance testers, front-end designer,  etc.  
+* Artistic designers
+* Music creators
+* Script writers
+* Project managers
+* Marketing specialists
+* Web designer
 
-As students get to know people in their community, they may even discover individuals who are willing to be mentors for your classroom in the days ahead.
+As your get to know people in your community, your can discover individuals who are willing to be mentors for your classroom in the days ahead.

```

Comment:
I believe this should read "As you get to know people in your community, you can discover..."

Reply:
Great catch